architecture-decision-records/0011-base25-encoding-for-consignment-reference.md

```diff
@@ -0,0 +1,88 @@
+# base25 encoding for consignment references
+
+**Date:** 2020-11-13
+
+## Context
+
+As part of the 'transfer summary' page, the user will see a 'consignment reference' to identify their consignment (which can be useful if a user needs to contact us about a specific consignment). We came to the decision that we liked the format:
+
+`TDR-YEAR-base25reference`
+
+`TDR` - since the consignment will be transferred using the TDR service
+
+`YEAR` - year the consignment is created (not completed)
+
+`base25reference` - created once the consignment is assigned an incremental ID, which will be encoded to a base25 alpha-numeric code.
```

Comment:
Is this part in upper case or lower case? (Unless it's mixed, but I don't think that's how the encoding algorithm works?)

Reply:
It's uppercase, I'll change that to reflect it! :+1: